src/main/java/seedu/address/logic/LogicManager.java

```diff
@@ -48,7 +48,8 @@ public CommandResult execute(String commandText) throws CommandException, ParseE
         try {
             storage.saveAddressBook(model.getAddressBook());
         } catch (IOException ioe) {
-            throw new CommandException(FILE_OPS_ERROR_MESSAGE + ioe, ioe);
+            String message = ioe.getMessage();
+            throw new CommandException(String.format(FILE_OPS_ERROR_MESSAGE, message), ioe);
```

Comment:
Looks like the code assumes every IOException is caused by _insufficient permissions_? I don't think that is a valid assumption.

Reply:
@damithc Agreed, I will display the insufficient permissions when catching the AccessDeniedException instead, and use a more generalised message for other possible I/O Exceptions.